src/Database/Table/Selection.php

```diff
@@ -227,6 +227,18 @@ public function fetch()
 
 
 	/**
+	 * Fetches single field.
+	 * @param  int
+	 * @return mixed|FALSE
+	 */
+	public function fetchField($column = 0)
+	{
+		$row = $this->fetch();
```

Comment:
ActiveRow should not support indexed keys, so $column default value should be NULL to allow `$table->select('xx')->fetchSingle()`